Mesh_3/include/CGAL/refine_mesh_3.h

```diff
@@ -170,11 +170,11 @@ class Insert_vertex_in_c3t3
  *                 of 1-dimensional exposed features.
  * \param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below.
  *           They control which optimization processes are performed
- *           and allow the user to tune the parameters of the optimization processes.
+ *           and enable the user to tune the parameters of the optimization processes.
  *           Individual optimization parameters are not described here as they are
  *           internal types (see instead the documentation page of each optimizer).
  *           For each optimization algorithm, there exist two global functions
- *           that allow to enable or disable the optimizer.
+ *           that enable to enable or disable the optimizer.
```

Comment:
"enable to enable". Isn't it a bit redundant?

Reply:
I think `allow` is correct or alternatively the more verbose `give the possibility`

Reply:
Allow is the opposite of forbid and Enable is the right word. ok for give the possibility.

Reply:
https://oilpatchwriting.wordpress.com/2010/10/13/enable-vs-allow/